packages/suite/src/views/onboarding/steps/SelectBackupType/OptionWithContent.tsx

```diff
@@ -160,4 +183,6 @@ export const OptionWithContent = ({
             </OptionText>
         </Option>
     );
+
+    return tooltip !== undefined ? <Tooltip content={tooltip}>{inner}</Tooltip> : inner;
```

Comment:
I would add `hasArrow`

Reply:
Maybe, shall we consider this to be default?

Reply:
💯